### ISSUE TITLE
Big bundle size

### ISSUE THREAD

**Issue by themre** [secretyouth/react-datez#10]
I looked at the size of this module and it's quite big: https://cost-of-modules.herokuapp.com/?p=react-datez@1.1.4

Perhaps it would be best to start trimming some stuff down or optimize code on some places?

**Comment by themre** [comment]
I see you're using express in here? why is this if it's a client side module? also, is everything linked to redux?

**Comment by secretyouth** [comment]
@themre I had a basic express server to run/dev it locally.

**Comment by themre** [comment]
but this needs to go away, otherwise it will install deeps also in other projects that will import this. what about redux. why do you need redux?

**Comment by themre** [comment]
i had the time to look a bit and saw that almost half of the packages are for your demos. usually you put demo app in separate and have there all those packages so they aren't loaded when someone uses your package. when I created my first npm package, I used https://github.com/insin/nwb

**Comment by secretyouth** [comment]
@themre, yeah it could definitely use a trim to bring the bundle size down.

I originally created this for redux forms as I needed for a project I was working on. I'll add it as an enhancement to bring the package size down and trim out the fat 👍

**Comment by sontek** [comment]
@secretyouth Any chance you've had time to trim this down? :)

**Comment by simontegg** [comment]
Its likely to be the dependency of [moment-js](https://bundlephobia.com/result?p=moment@2.21.0) that's fattening the bundle. 

I could have a go at swapping moment for [date-fns](https://date-fns.org/)
